### DISCUSSION TITLE
How to boot my os using Ventoy

### DISCUSSION THREAD

**Discussion by atlassoftware-cpu** [CosmosOS/Cosmos#2989] [Q&A]
So i made a great os. tried to run it couldn't run. searched on github, and found people saying: "disable VFS", and asked how to disable it. and never got an answer how. searched more and found out that it could work using ventoy. tried booting it using ventoy but couldn't. can anyone help?. Thanks

**Comment by 9xbt** [comment]
by "disabling the vfs" you literally dont initialize the vfs and dont use the filesystem

**Comment by 9xbt** [comment]
also "tried booting it using ventoy but couldn't" can you actually tell us specifically what happened

**Reply by atlassoftware-cpu** [reply]
is says Booting OSNAME.iso ... then stops. and the cursor stops flashing

**Reply by GeorgeFence** [reply]
Hmm... Does it enter to Limine?
